app/middlewares/userValidations.js

```diff
@@ -37,3 +39,12 @@ exports.validateLogin = (req, res, next) => {
     }
   });
 };
+
+exports.verifyAuthentication = (req, res, next) => {
+  if (req.headers.authorization) {
+    const token = req.headers.authorization.replace('Bearer ', '');
+    jwt.verify(token, config.common.session.secret) ? next() : res.status(401).send('Incorrect token');
```

Comment:
You got the idea of how to verify the session, but maybe you should check it a little bit more. You could check, for example, that the email encoded in the token is actually valid and exists in the database. What do you think?